cmake/O2AddTestWrapper.cmake

```diff
@@ -100,7 +100,7 @@ function(o2_add_test_wrapper)
 
   # For now, we enforce 3 max attempts for all tests.
   # No need to ignore time out, since we have 3 attempts
-  set(A_MAX_ATTEMPTS 3)
+  #PH set(A_MAX_ATTEMPTS 3)
```

Comment:
I do not understand what is the relation between the number of ctest attempts and ROOT 6.24?

Reply:
I mean, are we sure that now all tests are stable? We added the 3 attempts since some tests were failing randomly and the CI was red because of that quite often.

Reply:
It is very boring to wait 3 times for the same test to fail and to get the same long output three times. If a test is not stable, it should be fixed or disabled. I think there is no need to waste time with three attempts.

Reply:
ok, I agree with that, but then I think we need to be aggressive to immediately disable failing tests.
And then I think there are some geant tests with fail randomly, which we cannot fix, so perhaps at least for them we should maintain 3 attempts. @sawenzel can comment.

Reply:
If we want specific number of attempts, we should specify MAX_ATTEMPTS in the o2_add_test call and not change the value in the wrapper.

Reply:
Unless there is some urgency to fix the max_attempt right now, there is a PR in progress to get rid of some of our custom test cooking (https://github.com/AliceO2Group/AliceO2/pull/5916) (to be honest I've not been able to work on it lately, but I'll get back to it), so I guess the test policy change should not happen in this Root 6.24 PR.

Reply:
Ok, changed back with a reminder to take care later